endoreg_db/management/commands/load_requirement_data.py

```
@@ -12,8 +12,7 @@
     Event,
     LabValue,
     Finding,
-    FindingMorphologyClassificationChoice,
-    FindingLocationClassificationChoice,
+    FindingClassification, FindingClassificationChoice,
```

Ruff (0.12.2)

`endoreg_db.models.FindingClassification` imported but unused

Remove unused import: `endoreg_db.models.FindingClassification`

(F401)